docs/developer/eventing/sinks/creating-event-sinks/writing-event-sinks-with-golang/README.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "Writing an event sink using Golang"
+weight: 10
+type: "docs"
+showlandingtoc: "false"
+---
+
```

Comment:
this frontmatter isn't required anymore since me moved from hugo to mkdocs

Reply:
fixed: https://github.com/knative/docs/pull/4208/commits/b6cadbe876d44a25e58478fb8732f39eee14f712
Thank you!